README.md

```diff
@@ -1,7 +1,87 @@
-## AWS Serverless Metrics Publisher
+# AWS Serverless Metrics Publisher
 
-A metric publisher to batch metrics and send requests to CloudWatch for storage. 
+### A metric publisher to batch and publish metrics to CloudWatch.
+
+![Logo](Diagrams/MetricPublisherAppDiagram.png)
+
+## A Brief Explanation
+
+The Metric Publisher Application accepts metrics as input in JSON format,
+logs those metrics, and periodically gathers all new data from the Log Group,
+batches the new metrics together, and publishes them to CloudWatch. For more
+details, see "How to Deploy and Use" below.
+
+## How to deploy and Use
+
+### Initial Deployment
+
+To deploy the application, first fill in the sections under "Configure
+application parameters." The Application name can be left as it is:
+metric-publisher-app. The BatchingFrequencyInMinutes parameter is set at
+5 minutes by default, but can be modified to be as low as 1 minute, or as many
+as 60 minutes. This is the frequency that new metrics will be batched together
+and published to CloudWatch. The Namespace parameter also has a default value,
+but should be changed to something that better reflects the usage.
+
+Next, click "Deploy".
+
+Once the app has successfully deployed, click on "View CloudFormation stack" in
+the upper right hand corner.
+
+Under "Resources", click on the link under "Physical ID" corresponding to
+the "MetricPublisherRole" resource. Click the blue button labeled
```

Comment:
Under "Resources", click on the link under "Physical ID" corresponding to
the "MetricPublisherRole" resource and it redirects you to the IAM console.